packages/core/strapi/lib/services/entity-validator/__tests__/validators.test.js

```diff
@@ -20,6 +20,102 @@ describe('Entity validator', () => {
     fakeFindOne.mockReset();
   });
 
+  describe('String RegExp validaor', () => {
+    const fakeModel = {
+      kind: 'contentType',
+      modelName: 'test-model',
+      uid: 'test-uid',
+      privateAttributes: [],
+      options: {},
+      attributes: {
+        attrStringRequiredRegex: { type: 'string', required: false },
+        attrStringNotRequiredRegex: { type: 'string', required: false },
+      },
+    };
+
+    test('Empty string in required field', () => {
```

Comment:
Can you use a more descriptive test name?

Something like "_It fails the validation of an empty string for a required field_" so that we know exactly what the test does without having to look at the content.